lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context.cpp

```diff
@@ -31,10 +31,9 @@ extern "C" {
 
 namespace magma5g {
 extern task_zmq_ctx_t amf_app_task_zmq_ctx;
+extern std::unordered_map<imsi64_t, guti_and_amf_id_t> amf_supi_guti_map;
```

Comment:
we should limit the usage of `extern` declarations, especially for mutable types such as maps, if so please add a comment where is this initialized, as I see this pattern is also being used on `amf_recv.cpp` 

Ideally these all should be retrieved either by passed as arguments or through a `get` function, cc @ssanadhya

Reply:
This is a duplicate effort, these maps will be removed as part of PR (Fixing the global maps #11087).

Reply:
Please add a `TODO` comment to specify whis will be removed on #11087

Reply:
Addressed Review Comment